src/api/messages/ItemMessageInterface.ts

```diff
@@ -3,6 +3,9 @@ import { MarketplaceMessageInterface } from './MarketplaceMessageInterface';
 export interface ItemMessageInterface {
 
     // todo classes
+    hash: string;
+    marketId: number;
+    listingItemTemplateId: number;
     information: any;
```

Comment:
listingItemTemplateId can be different on the receiving end, there's no need for it to be here.
marketId would go to MarketplaceMessageInterface.